test/utils/ReceiptDataHelperSpec.scala

```diff
@@ -28,7 +28,7 @@ import play.api.mvc.{AnyContentAsEmpty, Cookie}
 
 class ReceiptDataHelperSpec extends BaseSpec {
 
```

Comment:
On line 52 of this file there's a long method which has caused a large scalastyle underline in my IDE. Would you mind adding `//scalastyle:ignore method.length` on the same line after the method declaration to surpress this.